app/controllers/api/v1/providers_controller.rb

```diff
@@ -16,10 +16,7 @@ class ProvidersController < API::V1::ApplicationController
       # - clock drift between servers
       def index
         # only return 2019 courses until rollover is supported
-        if params[:recruitment_year].present? && params[:recruitment_year] != '2019'
-          render json: [], status: :not_found
-          return
-        end
+        build_recruitment_cycle
```

Comment:
i thought you wanted the `before_action`

also the comments above is kinda not making sense any morish

Reply:
Done 👍